lib/oli_web/live/new_course/new_course.ex

```diff
@@ -525,15 +527,15 @@ defmodule OliWeb.Delivery.NewCourse do
             create_section(socket.assigns.live_action, assign(socket, changeset: changeset))
```

Comment:
I first tried converting just before create_section as you suggest!  But when the string-valued params go into a changeset, the local time string (which carries no tz indicator) winds up turned into a DateTime object which behaves as if it is a UTC time (the displayed string for it includes the Z indicator). Rather than doing time adjustment math, I settled for converting before validation and than localizing back if needed simply because the routines for doing that existed and were used in the edit_section view.

Reply:
Have changed to consider the time in that date validation. So now this allows a course that starts and ends on the same day.

Reply:
ok, makes sense! 😄 
Thanks for updating the date validation to consider the time 🙌